_includes/code/components/summary-box.html

```diff
@@ -1,4 +1,7 @@
 <h3 class="site-preview-heading">Default</h3>
```

Comment:
I don't think we need a heading or a "default" label here (because there are no non-default options)

Reply:
I removed the "default" header and updated the header in the component preview to be an `h3` in [3649ad4](https://github.com/uswds/uswds-site/pull/2608/commits/3649ad47fe9381563a18d0c5cf0f4daf00a94a8e). Noting that this does add the header to the in-page navigation link list. Is that ok?

Reply:
I discussed with Katie on Friday for a gut check, and we agreed that the "Key information" link that appears in the in-page nav looks awkward. Could we return to using an h4 for "Key information" and put a new h3 above it for the in-page nav? Instead of "Default" Katie and I think that "Component example" would be a better, more neutral option for the H3.

Reply:
@sarah-sch I've added "Default" headers to the component previews in Validation and Summary box, per our offline conversation in [c2bf69f](https://github.com/uswds/uswds-site/pull/2608/commits/c2bf69fe99300496d3fb025be8daa276d8a15532). Let me know if you need anything else!